catalog/USGS/USGS_NLCD_RELEASES_2021_REL_NLCD.jsonnet

```diff
@@ -0,0 +1,1019 @@
+local id = 'USGS/NLCD_RELEASES/2021_REL/NLCD';
+local subdir = 'USGS';
+//TODO (dpencosk) Add successor link to this dataset from: local successor_id = 'USGS/NLCD_RELEASES/2019_REL';
+local predecessor_id = 'USGS/NLCD_RELEASES/2019_REL';

```

Comment:
@dpencosk Use predecessor or successor link here to link to the 2019?

Reply:
predecessor would be correct I believe vs successor here

Reply:
Yes, predecessor is right.